aspnetcore/blazor/components/layouts.md

```diff
@@ -367,6 +367,61 @@ The following rendered HTML markup is produced by the preceding nested layout. E
 
 When routable components are integrated into a Razor Pages app, the app's shared layout can be used with the components. For more information, see <xref:blazor/components/prerendering-and-integration>.
 
+:::moniker range=">= aspnetcore-8.0"
+
+## Layout sections
+
+To control the content in layouts from Razor components, Blazor supports *sections* using the following built-in components:
+
+* `SectionOutlet`: Renders content provided by `SectionContent` components with matching `SectionId` or `SectionName` arguments. A `SectionOutlet` component can render the content of several `SectionContent` components.
```

Comment:
> A `SectionOutlet` component can render the content of several `SectionContent` components.

Not exactly. Here is how it works. 
`SectionOutlet` can render content of only one `SectionContent` with a matching `SectionName` or `SectionId`. Several `SectionContent`s can have same `SectionName` (`SectionId`).`SectionOutlet` will render content of the last rendered `SectionContent`.